tests/unit/src/test/scala/tests/DebugProtocolSuite.scala

```diff
@@ -35,35 +35,39 @@ object DebugProtocolSuite extends BaseLspSuite("debug-protocol") {
     } yield assertNoDiff(output, "Foo")
   }
 
-  testAsync("disconnect") {
-    for {
-      _ <- server.initialize(
-        s"""/metals.json
-           |{
-           |  "a": {}
-           |}
-           |/a/src/main/scala/a/Main.scala
-           |package a
-           |object Main {
-           |  def main(args: Array[String]) = { 
-           |    synchronized(wait())
-           |  }
-           |}
-           |""".stripMargin
-      )
-      debugger <- server.startDebugging(
-        "a",
-        DebugSessionParamsDataKind.SCALA_MAIN_CLASS,
-        new ScalaMainClass("a.Main", emptyList(), emptyList())
-      )
-      _ <- debugger.initialize
-      _ <- debugger.launch
-      _ <- debugger.configurationDone
-      _ <- debugger.disconnect
-      _ <- debugger.shutdown
-      output <- debugger.allOutput
-    } yield assertNoDiff(output, "")
-  }
+  // TODO: renable when this test is no longer flaky on Windows
+  // https://github.com/scalameta/metals/pull/1238/checks?check_run_id=370216834
+  // [1.321s][warning][os,thread] Failed to start thread - _beginthreadex failed (EACCES) for attributes: stacksize: default, flags: CREATE_SUSPENDED STACK_SIZE_PARAM_IS.
+  if (!isWindows)
```

Comment:
I think all of those test fail sometimes, but rather rare. I don't think we should remove them on Windows, since they are pretty neccessary for making sure debugging works on Windows.